spec/lib/error.spec.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+describe("lib/error", () => {
```

Comment:
Normally I wouldn't even bother testing something with this but since everything is mocked always, you sort of have to.